packages/core/lib/logger.js

```diff
@@ -1,5 +1,5 @@
 var winston = require('winston');
-var format = require('date-fns/format')
+var format = require('date-fns/format');
```

Comment:
Even if there is no ";" presenting at the end of each line, the code will still compile without throwing an error before?

Reply:
Yes. Semicolon is optional in Javascript and the code does compile and runs with or without it. However, as a best practice, it is advised to put use semicolons.